docs/guides/en/basic-functionality.md

```diff
@@ -166,8 +174,6 @@ $view->setTheme($theme);
 In a view, you can access the theme using the `getTheme()` method and manage assets as follows:
 
 ```php
-<?php
-
```

Comment:
`declare(strict_types=1);` can be removed also

Reply:
all occurrences in all files? Or just this one specifically?

Reply:
Just this one specifically.